src/pl/plajer/pinata/Main.java

```diff
@@ -20,7 +20,7 @@
 
     private static Main instance;
     private final int MESSAGES_FILE_VERSION = 10;
-    private final int CONFIG_FILE_VERSION = 5;
+    private final int CONFIG_FILE_VERSION = 6;

```

Comment:
No need changing it.

Reply:
Ooops.. I'm sorry, I did not know.

Reply:
I should say: "No need to change it if you added only comment to the config.yml" :)